package.json

```diff
@@ -18,19 +18,19 @@
     "clean:es": "rimraf es",
     "build:es": "cross-env BABEL_ENV=es babel src --out-dir es --copy-files",
     "build": "yarn clean:lib && yarn build:lib && yarn clean:es && yarn build:es",
-    "test": "tape -r esm \"test/**/*-test.js\"",
+    "test:lib": "yarn build:lib && tape \"test/**/*-test.js\"",
```

Comment:
Let's rename to `test` for now or change `check` (which uses `test` not `test:lib`). We're not doing CI yet, so this is all slightly premature (but isn't that big of a deal).